apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -516,20 +520,32 @@ export const NewHomeScreen = (): ReactElement => {
     if (!isLoadedPCDs) return;
 
     const maybeExistingFolder = params.get("folder");
-    if (
-      maybeExistingFolder &&
-      collection
-        .getAllFolderNames()
-        .includes(replaceDotWithSlash(decodeURI(maybeExistingFolder)))
-    ) {
-      if (!showPodsList) {
-        dispatch({
-          type: "set-bottom-modal",
-          modal: { modalType: "pods-collection" }
-        });
+    if (maybeExistingFolder) {
+      // Check if folder matches any zapp name (case insensitive)
+      const zappEntry = Object.entries(appConfig.embeddedZapps).find(
```

Comment:
when i have nested folder, i.e "collections/frogcrypto", the url `localhost:3000/#/?folder=Collections%20%c2%b7%20FrogCrypto` goes to the pod list not the frog crypto

Reply:
yep, that is expected. localhost:3000/#/?folder=Collections%20%c2%b7%20FrogCrypto should go to pod collection modal, scrolled to the right folder, while localhost:3000/#/?folder=frogcrypto should open the frogcrypto zapp.